src/CalcViewModel/StandardCalculatorViewModel.cpp

```diff
@@ -488,7 +488,7 @@ void StandardCalculatorViewModel::HandleUpdatedOperandData(Command cmdenum)
         m_selectedExpressionLastData = L"";
         if (ch == 'x')
         {
-            temp = L'\0';
+            temp[0] = L'\0';
```

Comment:
While your're changing this, let's remove the new and replace it with something better.